client/components/field-error/index.js

```diff
@@ -1,17 +1,22 @@
 import React, { PropTypes } from 'react';
 import sanitizeHTML from 'lib/utils/sanitize-html';
 import Gridicon from 'gridicons';
+import classNames from 'classnames';
 
-const FieldError = ( { text } ) => {
+const FieldError = ( { text, type = 'input-validation' } ) => {
 	return (
-		<div className="form-input-validation is-error">
+		<div className={ classNames(
+			'is-error',
+			[ `form-${ type }` ]
```

Comment:
👍 I'm using `form-` as the prefix since this is part of a form. Would prefer to leave a more generic-use-case refactoring for when it's needed.